kube/kd/deployment.yaml

```diff
@@ -266,18 +266,23 @@ spec:
                 secretKeyRef:
                   name: hocs-casework
                   key: rds_password
-            - name: AUDIT_TOPIC_NAME
+            - name: AWS_SNS_AUDIT_SEARCH_TOPIC_NAME
```

Comment:
Oh look, we're deploying otherwise breaking changes to the deployment along with the code that needs it :)